src/Components/NoLokiSplash.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { useStyles2, useTheme2 } from '@grafana/ui';
+import SVG from 'react-inlinesvg';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+
+export const NoLokiSplash = () => {
+  const styles = useStyles2(getStyles);
+  const theme = useTheme2();
+  return (
+    <div className={styles.wrap}>
+      <div className={styles.graphicContainer}>
+        <SVG
+          src={
+            theme.isDark
+              ? `/public/plugins/grafana-lokiexplore-app/img/grot_loki.svg`
+              : `/public/plugins/grafana-lokiexplore-app/img/grot_loki.svg`
```

Comment:
I did look at this for 2minutes, but don't see a difference. Is this needed?

Reply:
No 🤦